---
template: blog_post.html
title: Markers, Markers Everywhere - Will they Kill Me? Should I Care?
description: Reporting of SNVs identified in GWAS studies in news
date: 2018-03-22
authors:
  - "@mbaudis"
hide:
  - navigation
  - toc
---

This can be very brief: Since the arrival of genomic arrays aimed at mapping Single Nucleotide Polymorphisms (SNP), and later sequencing based variation profiling, there has been a deluge of (more or less) well designed studies to look for a statistical correlation between the occurrence of SNP variants at genomic loci, and (more or less) well defined traits or phenotypes.

<!--more-->

Such studies can deliver interesting information about phenotypic heritability, or even may lead to the identification of genes or genome elements causally related to specific traits or disease risks.

However, a common problem in the reporting of genotype-phenotype associations from GWAS studies through news outlets or even the PR departments of universities or organisations involved in such studies  lies in the conflation of association and causality. So, some guidelines:

GWAS report, well, associations of genomic variants (usually SNV, occasionally CNV) with phenotypes/traits/disease risk etc. These markers (even if holding up in confirmation studies) most likely have no causality with respect to the phenotypic quality. They are markers because of a statistical probability to co-segregate with the observed phenotype/trait, due to a low genomic distance (i.e. low probability of recombination) from a possibly causative genomic variant.

Since such markers are just this, they **cannot** kill you, make you live longer, lead to a happy life …
Some more details about GWAS can e.g. be found in this older but still valid [blog post by Jeff Barrett](https://genomesunzipped.org/how-to-read-a-genome-wide-association-study/).

So: Even if a genome marker (i.e. specific SNP) has a high concordance with a given trait (such as longevity – and there are way too many to be concerned about a single one) – markers are just that. And when reporting about them, a language of “genome variant showing statistical association with” instead of “responsible for” has to be used.

Now about effect size … But I wanted to stay brief.
